appveyor.yml

```diff
@@ -2,61 +2,30 @@ branches:
   only:
     - master
 environment:
+  nodejs_version: "6"
   github_access_token:
     secure: VMFbecLLHzDq/09YDPbcM0VDDSwwgY57vr5GXK6cZZ4Ti/Xs5RZoylzV8MMr1350
 
-build_script:
-  - ps: |
-      $ErrorActionPreference = "Stop"
-      $response = Invoke-WebRequest ("https://api.github.com/repos/" + $env:APPVEYOR_REPO_NAME + "/releases/latest") -Timeout 10
-      if($response -ne $null -and $response.StatusCode -ne 200)
-      {
-        $host.SetShouldExit(1)
-      }
-
-      $unzip_path = "C:\projects\release"
-      $unzip_file = $unzip_path + "zip"
-      Invoke-WebRequest ($response.Content | ConvertFrom-Json).zipball_url -OutFile $unzip_file
-      7z x $unzip_file -o"$unzip_path"
-      $src = Join-Path (ls $unzip_path -dir | select -First 1 | % { $_.FullName }) "src"
-      $global:output = Join-Path $unzip_path "output"
-      ni $global:output -type dir
-      $pattern = "\d(.\d)*"
-
-      foreach($folder in (ls $src -dir))
-      {
-        foreach($module in (ls $folder.FullName -dir | % { $_.FullName }))
-        {
-          # find out docs in help folder and version in .psd1 file 
-          $help = ls $module -dir -Recurse | ? { $_.Name -eq "help" } | select -First 1 | % { $_.FullName }
-          if($help -eq $null)
-          {
-            continue
-          }
-          $psd1= ls $module | ? { $_.extension -eq ".psd1" } | select -First 1 | % { $_.FullName }
-          if($psd1 -eq $null)
-          {
-            continue
-          }
-          if((gc $psd1 | Out-String) -notmatch "ModuleVersion\s*=\s*'$pattern'")
-          {
-            continue
-          }
-          if($matches[0] -match $pattern)
-          {
-            robocopy $help "*.md" (Join-Path $global:output $folder | Join-Path -ChildPath (gi $psd1).BaseName | Join-Path -ChildPath ("v" + $matches[0]))
-          }
-        }
-      }
-test: off
-shallow_clone: true
-on_success:
+before_build:
+  - set path=%programfiles(x86)%\Microsoft SDKs\TypeScript\2.2;%path%
```

Comment:
This is making all kinds of assumptions about the environment it is running in.  What are the required environment variables, and why is it assumign it is runnign on a 64-bit system?